src/picasso-components/react-components/navigation-panel.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { IconButton } from '@material-ui/core';
+import SVGIcon from './icons/SVGIcon';
+import NUMBERS from '../../constants/numbers';
+import ICONS from './icons';
+
+const buttonWidth = NUMBERS.NAVIGATION_PANEL.GRID_WIDTH;
+const padding = NUMBERS.NAVIGATION_PANEL.PADDING;
+
+const useStyles = (rtl) =>
+  makeStyles(() => {
+    const horizontalDistance = rtl ? 'left' : 'right';
+    return {
+      onFocus: {
+        '&:focus': {
+          background: 'rgba(240, 240, 240, 0.5)',
+        },
+      },
+      common: {
+        position: 'absolute',
+        borderRadius: '2px',
+        padding: '0',
+        width: `${buttonWidth}px`,
+        height: `${buttonWidth}px`,
+      },
+      UP: { top: '0px', [horizontalDistance]: `${buttonWidth}px` },
+      LEFT: { top: `${buttonWidth}px`, [horizontalDistance]: `${rtl ? 0 : 2 * buttonWidth}px` },
+      HOME: { top: `${buttonWidth}px`, [horizontalDistance]: `${buttonWidth}px` },
+      RIGHT: { top: `${buttonWidth}px`, [horizontalDistance]: `${rtl ? 2 * buttonWidth : 0}px` },
+      DOWN: { top: `${2 * buttonWidth}px`, [horizontalDistance]: `${buttonWidth}px` },
+      ZOOM_IN: { top: `${3 * buttonWidth}px`, [horizontalDistance]: `${buttonWidth}px` },
+      ZOOM_OUT: { top: `${4 * buttonWidth}px`, [horizontalDistance]: `${buttonWidth}px` },
+    };
+  });
+
+const NavigationButton = ({ ...rest }) => {
+  const classes = useStyles(rest.rtl)();
+  return (
+    <IconButton className={`${classes.onFocus} ${classes.common} ${classes[rest.buttonName]}`} onClick={rest.callback}>
```

Comment:
Maybe better to remove prefix button for all. It makes the line so long and we kind know they are buttons. e.e, buttonName, buttonWidth

Reply:
done!